templates/editor/main-partial.html

```diff
@@ -875,6 +875,9 @@ <h3 class="cheatsheet-title">Storage shortcuts</h3>
   loadingMask.style.opacity = 0;
   window.setTimeout(function () {loadingMask.parentNode.removeChild(loadingMask);}, 600);
   pskl.app.init();
+  $.subscribe('PISKEL_DESCRIPTOR_UPDATED', function(a) {
```

Comment:
My fault for adding this to the source control, but this file is automatically generated from https://github.com/juliandescottes/piskel, so we can't really change it.

This probably means we first need Piskel to notify its host (piskel-website) when it is loaded. Piskel-website could then add the event listener you added here. I guess it's best to leave this commit out of this PR for now?